action.yml

```
@@ -17,78 +17,75 @@ inputs:
     required: false
-    description: "Format of the output file"
+    description: "Output file format"
   output:
     required: false
-    description: "Send analysis results to an output file"
+    description: "Path to a file to save the analysis results"
   directory:
     required: false
-    description: "The directory to be analyzed"
+    description: "Directory to analyze"
   parallel:
     required: false
     description: "Number of tools to run in parallel"
   max-tool-memory:
     required: false
-    description: "Maximum of allowed memory for each tool execution"
+    description: "Maximum allowed memory for each tool (bytes)"
   max-allowed-issues:
     required: false
     default: "2147483647"
     description: "Maximum number of issues allowed for the analysis to succeed"
   tool:
     required: false
-    description: "The name of the tool to analyze the code"
+    description: >-
+      Only run a specific tool or tool category (metrics, issues, duplication).
+      For a full list, see https://docs.codacy.com/repositories-configure/codacy-configuration-file/#which-tools-can-be-configured-and-which-name-should-i-use
   tool-timeout:
     required: false
-    description: "Maximum time each tool has to execute"
+    description: "Tool execution timeout (e.g. 15minutes, 1hour)"
   upload:
     required: false
     description: "Upload analysis results to Codacy"
   upload-batch-size:
     required: false
-    description: "Maximum number of results in each batch to upload to Codacy"
+    description: "Maximum number of uploaded results per batch"
   fail-if-incomplete:
     required: false
     description: "Fail the analysis if any tool fails to run"
   allow-network:
     required: false
-    description: "Allow network access for tools"
+    description: "Allow the tools to access the network"
   force-file-permissions:
     required: false
     description: "Force files to be readable by changing the permissions before running the analysis"
   gh-code-scanning-compat:
     required: false
     description: >-
-      Reduce issue severity by one level, for non-security issues, for compatibility with GitHub's code scanning feature.
-      This option will only have an effect when used in conjunction with '--format sarif'.
+      Reduce issue severity by one level for non-security issues, for compatibility with GitHub's code scanning feature.
+      This option only has an effect when used with 'format: sarif'.
   run-docker-tools:
     required: false
     default: "true"
     description: >-
-      Run dockerized tools. Possible values (true|false)
-      Check the full list in https://docs.codacy.com/repositories-configure/codacy-configuration-file/#which-tools-can-be-configured-and-which-name-should-i-use
+      Run all dockerized tools supported by Codacy.
+      For a full list, see https://docs.codacy.com/repositories-configure/codacy-configuration-file/#which-tools-can-be-configured-and-which-name-should-i-use
```

Small additional clarification:

```suggestion
      For the full list of tools, see https://docs.codacy.com/repositories-configure/codacy-configuration-file/#which-tools-can-be-configured-and-which-name-should-i-use
```